docs/get-started/setup.md

```diff
@@ -60,15 +51,15 @@ A common frontend pattern is for components to assume that they render in a cert
 
 Use [decorators](../writing-stories/decorators.md) to “wrap” every story in the necessary context providers. [`.storybook/preview.js`](../configure/overview.md#configure-story-rendering) allows you to customize how components render in Canvas, the preview iframe. In this decorator example, we wrap every component rendered in Storybook with `ThemeProvider`.
 
-```js
-// .storybook/preview.js
+<!-- prettier-ignore-start -->
 
-import { ThemeProvider } from 'styled-components';
-export const decorators = [
-  (Story) => <ThemeProvider theme="default"><Story/></ThemeProvider>,
-];
+<CodeSnippets
+  paths={[
+    'common/storybook-preview-with-styled-components-decorator.js.mdx',
```

Comment:
This is definitely react-specific!

Reply:
the snippet is now in the correct place and the page is updated. Thanks for catching that